### PR TITLE
add design and functionality of Cart page

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add code for the design and functionality of Cart page

In this PR, **872 additions and 56 deletions** lines of code
- Some files contain only 5 to 6 lines of code
- If you don't prefer you can avoid to review CSS and backend related files

### Please review this PR and give your feedback

---
```javascript
email: johndoe@gmail.com
password: johnDoe123
```

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 import {
+  Cart,
+  Home,
+  NotFound,
+  Products,
+  SignIn,
+  SignUp,
+  WishList,
+} from "pages";
+import {
```
Comment (arjun-computer-geek) [inline src/App.js:13]: This is great approach. I liked it.

File: src/components/card/cart-product-card/CartProductCard.jsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          handleClick={handleWishListClick}
+          btnText={`${isInWishList ? "Remove from" : "Move to"} Wishlist`}
+          btnType="solid"
+        />
+      </div>
+    </div>
+  );
+};
+
+CartProductCard.propTypes = {
```
Comment (arjun-computer-geek) [inline src/components/card/cart-product-card/CartProductCard.jsx:100]: Wow bro you are using proptype great

**Review by arjun-computer-geek — APPROVED** [review]
Great code of conduct. Good importing and exporting all well arranged. Easy to readable. But one suggestion leave some comments what are you doing. It will be great. Other then that great job himnashu👍👍👍👍😁.

**Review by yash9263 — APPROVED** [review]

**Review by yash9263 — APPROVED** [review]

**Comment by yash9263** [comment]
code looks clean and good creating utils functions for common operations.

**Comment by hsnice16** [comment]
Thank you for reviewing.

**Pull request merged**